### PR TITLE
4.6 merge master

### PR DESCRIPTION
**Pull request by adigrabow**

### CODE REVIEW

File: cosmo_tester/framework/test_hosts.py [hunk 1]
```diff
@@ -254,6 +254,27 @@ def upload_necessary_files(self):
             fabric_ssh.sudo('chmod 440 {key_file}'.format(
                 key_file=REMOTE_PRIVATE_KEY_PATH,
             ))
+            self.enter_sanity_mode()
+
+    def enter_sanity_mode(self):
+        """
+        Test Managers should be in sanity mode to skip Cloudify license
+        validations.
+        :return:
```
Comment (tyacbovi) [inline cosmo_tester/framework/test_hosts.py]: Can remove this

File: cosmo_tester/framework/test_hosts.py [hunk 2]
```diff
@@ -432,7 +453,7 @@ def _create_config_file(self):
         config_file.write_text(install_config_str)
         return config_file
 
-    def bootstrap(self):
+    def bootstrap(self, db=False):
```
Comment (tyacbovi) [inline cosmo_tester/framework/test_hosts.py]: What "db" means?

Comment (adigrabow) [reply cosmo_tester/framework/test_hosts.py]: Whether or not this is an external DB machine (if it is we do not need to enter sanity mode)

File: cosmo_tester/framework/test_hosts.py (hunk 1 above)
Comment (adigrabow) [reply cosmo_tester/framework/test_hosts.py]: Why? I think it's a good comment :(

Comment (tyacbovi) [reply cosmo_tester/framework/test_hosts.py]: I meant only line 263

File: cosmo_tester/framework/test_hosts.py (hunk 2 above)
Comment (tyacbovi) [reply cosmo_tester/framework/test_hosts.py]: so maybe change it to "with_db" or "db_contained"

**Pull request merged**